### ISSUE TITLE
NTRs for lifestyles

### ISSUE THREAD

**Issue by pbuttigieg** [microbialphenotypes/OMP-ontology#208]
Greetings all! We need classes for microbial lifestyles like "free-living", "biofilm-associated", "substrate-bound", and "host-associated" - these aren't exactly phenotypes (more the result of a combination of phenotypes), but OMP seems like the best target available. Would you be able to cook up these semantics?

**Comment by dsiegele** [comment]
I think the requested terms qualify as phenotypes. I will work on it this week.

**Comment by pbuttigieg** [comment]
Many thanks, shall I propose defs?

**Comment by dsiegele** [comment]
Proposing definitions would be helpful.

**Comment by pbuttigieg** [comment]
Something along the lines of...

'free-living phenotype' =def. "A phenotype which is the summative outcome
of multiple phenotypes that, acting together, typically provide an organism
some advantage while competing for niche space in a fluid medium,
unattached to other organisms or non-living, solid material entities."

Comment: "May include the expression of flagella, sensing apparati used for
photo- and chemotactic behaviour, and quorum sensing pathways."

'attached phenotype' =def. "A phenotype which is the summative outcome of
multiple phenotypes that, acting together, typically provide an organism
some advantage while competing for niche space while it is physically anchored
it to some portion of solid material."


Comment: "May include the expression of pilli or other structures used for
attachment, antibiotic compounds, extracellular matrix remodelling
proteins, and the like. "


We could add to the comments that these definitions are not exclusively
used for microbial organisms.

You can probably see why I'm inclined to call these lifestyles or
behaviours rather than phenotypes, but I suppose they could be understood
as compound phenotypes. Perhaps we can add an editor note raising this
concern?

Also, while these phenotypes have evolved to confer some eecological
advantage, they may never do so. Thus the somewhat fuzzy "typically " in
the defs. Perhaps there's a better way to define these by referencing a
population of microbes of the same type, rather than a single organism.

You can use ENVO's solid, liquid, and gaseous material classes to
axiomatise these. They are driven by inference using PATO qualities, so
should be quite useful and future proof.

**Comment by dsiegele** [comment]
Thanks, Pier. That's a big help.

**Comment by pbuttigieg** [comment]
Hi @dsiegele, would these be released soon? We have some users that would like to use these very soon. We could create temporary classes in ENVO and obsolete these once you make a release, but I'd prefer to avoid this.

**Comment by dsiegele** [comment]
Hi Pier,  I apologize for the delay. I was away for 2 weeks for a family emergency, but am back and will work on these now.  Debby

**Comment by pbuttigieg** [comment]
Hi Debby - not to worry, please don't feel pressured; we can make the temporary classes if you need more time.

**Comment by dsiegele** [comment]
I found the following in Prokaryotes (2006) 2:3–15 DOI: 10.1007/0-387-30742-7_1
"Because of the extremely small size of most prokaryotic organisms, the limits on what is meant by the terms planktonic and sessile require definition. According to the Oxford English Dic- tionary, planktonic refers to “drifting or floating organic life found at various depths in the ocean or fresh water.” At the micrometer level, a planktonic habitat for prokaryotes can also encompass water films around soil particles, saliva in the mouth, fluids in the intestinal lumen, serum in blood vessels, and urine in the bladder and urinary tract. Sessile, on the other hand, means “immediately attached, without a footstalk.” _Again, **one can extend this definition** to include those prokaryotes directly adhering to surfaces, those attaching by means of a holdfast at the end of a prostheca (e.g., Caulobacter), those embedded in biofilms developing as a result of extracellular polymer production by bacteria colonizing surfaces, and those colonizing mucus excreted by higher organisms (as in the gastrointestinal tract and the mucigel of plant roots)_." 

Does your conception of the attached lifestyle phenotype extend to microbes colonizing mucus? The human microbiome includes both microbes that attach to the surface of host cells and microbes that grow in mucus without directly contacting the host. Should the definition make a distinction between these two types of growth? I don't of growing in mucus as being "physically anchored to some portion of solid material," but to a microbe mucus is probably solid.

**Comment by pbuttigieg** [comment]
> Does your conception of the attached lifestyle phenotype extend to microbes colonizing mucus? 

Good catch and thanks for the cross-ref. We can re-tune and simplify the defs to:

'free-living phenotype' =def. "A compound phenotype which grants a microbe the ability to live and compete for niche space in a fluid medium, unattached to and not embedded in other living or non-living, solid material entities."

Comment: "May include the expression of flagella, sensing apparati used for photo- and chemotactic behaviour, and quorum sensing pathways. These component phenotypes are not necessarily exclusive to free-living organisms."

'attached phenotype' =def. "A compound phenotype which grants an organism
the ability to live and compete for niche space while it is physically anchored to or embedded in some portion of solid material."

exact syn: 'embedded phenotype'

Comment: "May include the expression of pilli, holdfasts, or other structures used for
attachment, antibiotic compounds, extracellular matrix remodelling proteins, and the like. May also include those phenotypes which enable a microbe to compete while embedded in biofilms or substances produced by other organisms."

If not present already, I think it would be a good idea to think about adding something like "compound phenotype" (as an inferred class) to classify phenotypes which have other phenotypes as "parts" or similar.

**Comment by dsiegele** [comment]
OMP:0007645 | microbial lifestyle phenotype
def: A microbial phenotype related to the lifestyle(s) of a microbe. Lifestyle phenotypes are compound phenotypes, i.e. they are the result of multiple phenotypes acting together.

.....OMP:0007647 | attached lifestyle phenotype
def: _A microbial lifestyle phenotype relating to whether a microbe has the ability to live and compete for resources while physically anchored to or embedded in some portion of solid material._
Comment: _Attached lifestyle phenotypes are compound phenotypes, i.e. they are the result of multiple phenotypes acting together, and may include the expression of pili or other structures used for attachment, antibiotic compounds, extracellular matrix remodelling proteins, and the like._

..........OMP:0007667 | absence of a free-living lifestyle
def:_A attached lifestyle phenotype where a microbe lacks the ability to live and compete for resources while physically anchored to or embedded in some portion of solid material._
..........OMP:0007665 | altered attached lifestyle
def:_An attached lifestyle phenotype where the ability of a microbe to live and compete for resources while physically anchored to or embedded in some portion of solid material is altered relative to a designated control._
..........OMP:0007648 | presence of an attached lifestyle
def:_A attached lifestyle phenotype where a microbe has the ability to live and compete for resources while physically anchored to or embedded in some portion of solid material.

.....OMP:0007646 | free-living lifestyle phenotype
def:_A microbial lifestyle phenotype relating to whether a microbe has the ability to live and compete for resources in a fluid medium, unattached to and not embedded in other living or non-living, solid material._
..........OMP:0007664 | absence of an attached lifestyle
def:_A free-living lifestyle phenotype where a microbe lacks the ability to live and compete for resources in a fluid medium, unattached to and not embedded in other living or non-living, solid material._
..........OMP:0007668 | altered free-living lifestyle
def:_A free-living lifestyle phenotype where the ability of a microbe to live and compete for resources in a fluid medium, unattached to and not embedded in other living or non-living, solid material is altered relative to a designated control._
..........OMP:0007666 | presence of a free-living lifestyle
def:_A free-living lifestyle phenotype where a microbe has the ability to live and compete for resources in a fluid medium, unattached to and not embedded in other living or non-living, solid material._
